requirements.txt

```diff
@@ -0,0 +1,6 @@
+- --no-binary :all:
```

Comment:
May need to also add ``--no-build-isolation`` also, but let's try this first as, after that, I'm not sure what to do but 😭